gradio/processing_utils.py

```diff
@@ -192,6 +192,7 @@ def audio_from_file(filename, crop_min=0, crop_max=100):
             print(
                 "Please install `ffmpeg` in your system to use non-WAV audio file formats."
             )
+        return 0, np.empty(0)
```

Comment:
Thanks @FelixDombek ! Wouldn't it be better to raise the exception in this case? The user would get an empty audio and not know why.

Reply:
You're probably right. If I replace the line with a simple `raise` , I get:
```
C:\ai\stable-diffusion-webui\venv\lib\site-packages\pydub\utils.py:198: RuntimeWarning: Couldn't find ffprobe or avprobe - defaulting to ffprobe, but may not work
  warn("Couldn't find ffprobe or avprobe - defaulting to ffprobe, but may not work", RuntimeWarning)
Traceback (most recent call last):
  File "C:\ai\stable-diffusion-webui\venv\lib\site-packages\gradio\routes.py", line 284, in run_predict
    output = await app.blocks.process_api(
  File "C:\ai\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 980, in process_api
    inputs = self.preprocess_data(fn_index, inputs, state)
  File "C:\ai\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 891, in preprocess_data
    processed_input.append(block.preprocess(inputs[i]))
  File "C:\ai\stable-diffusion-webui\venv\lib\site-packages\gradio\components.py", line 1845, in preprocess
    sample_rate, data = processing_utils.audio_from_file(
  File "C:\ai\stable-diffusion-webui\venv\lib\site-packages\gradio\processing_utils.py", line 193, in audio_from_file
    audio = AudioSegment.from_file(filename)
  File "C:\ai\stable-diffusion-webui\venv\lib\site-packages\pydub\audio_segment.py", line 728, in from_file
    info = mediainfo_json(orig_file, read_ahead_limit=read_ahead_limit)
  File "C:\ai\stable-diffusion-webui\venv\lib\site-packages\pydub\utils.py", line 274, in mediainfo_json
    res = Popen(command, stdin=stdin_parameter, stdout=PIPE, stderr=PIPE)
  File "C:\Program Files\Python310\lib\subprocess.py", line 969, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Program Files\Python310\lib\subprocess.py", line 1438, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified.
```
The file that isn't found is actually ffprobe. The first line also tells us it's missing. Just putting this into PATH makes the crash go away. (I wonder what is going on here, since Riffusion works just fine without this huge dependency except for this one (circumventable) crash. Maybe it can be fixed to avoid this call path altogether, at least I don't understand it's purpose.) 

It looks as if the code here was written to look for this condition, but the error message is different on Windows and doesn't contain `"ffprobe"`. So Gradio doesn't output the useful hint (line 193). 

**Should we just make the output unconditional? Something like "Make sure ffprobe is in your PATH". WDYT?**

Reply:
I made a commit to that effect, let me know what you think. :)

Reply:
Thank you for the change! Maybe it's better if the exception you raise contains the helpful text you added as opposed to the print. I'm worried that text will be lost in the console when the original exception is raised. Let me know what you think.